cmd/config/virt-capacity-benchmark/virt-capacity-benchmark.yml

```diff
@@ -0,0 +1,207 @@
+{{- $kubeBurnerFQDN := "kube-burner.io" -}}
+{{- $testName := "capacity-benchmark" }}
+{{- $nsName := .testNamespace -}}
+{{- $vmCount := .vmCount -}}
+{{- $vmName := $testName -}}
+{{- $sshPublicKeySecretName := $testName -}}
+{{- $rootVolumeSize := 6 -}}
+{{- $dataVolumeSize := 1 -}}
+{{- $volumeLabelKey := (list $testName "." $kubeBurnerFQDN "/volume-type") | join "" -}}
+{{- $volumeLabelValueRoot := "root" -}}
+{{- $volumeLabelValueData := "data" -}}
+{{- $jobCounterLabelKey := (list $testName "." $kubeBurnerFQDN "/counter") | join "" -}}
+{{- $jobCounterLabelValue := (list "counter-" (.counter | toString )) | join "" -}}
+{{- $testNamespacesLabelKey := (list $kubeBurnerFQDN "/test-name") | join "" -}}
+{{- $testNamespacesLabelValue := $testName -}}
+{{- $metricsBaseDirectory := $testName -}}
+---
+global:
+  measurements:
+  - name: vmiLatency
+
+metricsEndpoints:
+- indexer:
+    type: local
+    metricsDirectory: "./{{ $metricsBaseDirectory }}/iteration-{{ .counter | toString }}"
+
+jobs:
+# Run cleanup only when counter is 0
+{{ if eq (.counter | int) 0 }}
+- name: start-fresh
+  jobType: delete
+  waitForDeletion: true
+  qps: 5
+  burst: 10
+  objects:
+  - kind: Namespace
+    labelSelector:
+      {{ $testNamespacesLabelKey }}: {{ $testNamespacesLabelValue }}
+{{ end }}
+
+- name: create-vms
+  jobType: create
+  jobIterations: 1
+  qps: 20
```

Comment:
curious why in the first job you have the CNV default of 5/10 for QPS/Burst -- but here and further down you default to 20/20?

Reply:
TBH I didn't put too much thought into it. Especially since most of the jobs are pretty small. So, not sure it even matters. Which values would you recommend here?